llvm/docs/CommandGuide/llvm-cxxfilt.rst

```diff
@@ -52,6 +52,11 @@ OPTIONS
 
   Do not demangle function parameters or return types.
 
+.. option:: --quote, -q
+
+  Add `"` `"` around demangled function symbols, typically to keep LLVM/MLIR
+  assembly files with `@<symbols>` valid. Do not quote already quoted symbols.
```

Comment:
I can imagine other cases where the quoting behaviour might be useful, so I'd be inclined to drop the "typically to keep ..." part of this sentence.